prestoadmin/util/version_util.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Stuff to handle version ranges.
+"""
+
+import re
+
+TD_VERSION = re.compile(r'^\d+t$')
+ANCIENT_TAGGED_VERSION = re.compile(r'^\d+t?-[A-Z]+$')
+
+
+def split_version(version_string):
+    return version_string.strip().split('.')
```

Comment:
Because if it ever changes, we'll have n places to change the code instead of 1.